input/community/speakers/geoffrey-huntley.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: Geoffrey Huntley
+Location: [Brisbane|Sydney|Melbourne], Australia
```

Comment:
```suggestion
Location: "[Brisbane|Sydney|Melbourne], Australia"
```

Adding quotes in an attempt to avoid the yaml parsing error that's being thrown.